helpers/handlebars/middleware.js

```diff
@@ -114,7 +115,15 @@ module.exports = (req, res, next) => {
 				link: '/files/shared/',
 			},
 		],
-	}, {
+	},
+	{
+		name: res.$t('global.headline.files'),
+		testId: 'Dateien',
+		icon: 'folder-open',
+		link: '/new-files/',
```

Comment:
since routes are relatively difficult to change, I would like to avoid the word "new" in a route. think of Windows NT, which is not very new technology anymore.

How about "/collaborative-files/" or "files-collaborative" or something? or /filesv2/ or something...

Reply:
I was hoping that suggestions would come :) 
I even called it files2 at the beginning.. but that didn't work becasue of this (https://github.com/hpi-schul-cloud/nuxt-client/blob/cca4756ad18bd01640a4671422be61168ecec0a1/src/layouts/legacyLoggedIn.vue#L99-L99) and I didn't dare to do that